pkg/cmd/controller/apiserver_authenticator.go

```diff
@@ -24,8 +25,8 @@ func newRemoteAuthenticator(tokenReview authenticationv1client.AuthenticationV1I
 	// TODO audiences
 	TokenAccessReviewTimeout := 10 * time.Second
 	tokenAuthenticator, err := webhooktoken.NewFromInterface(tokenReview, nil, *webhooktoken.DefaultRetryBackoff(), TokenAccessReviewTimeout, webhooktoken.AuthenticatorMetrics{
-		RecordRequestTotal:   noopMetrics{}.RequestTotal,
-		RecordRequestLatency: noopMetrics{}.RequestLatency,
+		RecordRequestTotal:   authenticatorfactory.RecordRequestTotal,
+		RecordRequestLatency: authenticatorfactory.RecordRequestLatency,
```

Comment:
yes if we think the metrics are valuable :)

The delegated AuthN/Z uses the same metrics since OCM is not using the generic API library we need to wire them manually.